### DISCUSSION TITLE
Packaging the Core of libraries of this tool as Nuget

### DISCUSSION THREAD

**Discussion by CarlosSardo** [microsoft/Tools-for-Health-Data-Anonymization#149] [Ideas]
Hi,

Would you see any added value by package and publish the core libraries of this tool as Nuget, so that other developers could more easily use the Anonymization engine in their custom built .net apps/services?

Thanks,
Carlos

**Comment by ginalee-dotcom** [comment]
Hi @CarlosSardo, yes we should be able to package and publish the tool libraries as a nuget package, since that is what we usually do with some of our OSS projects to be consumed by the [FHIR server](https://github.com/microsoft/fhir-server). For example, we package our anonymized export feature as well as our converter feature. 

Pulling in @qiwjin for a confirmation.

**Comment by Erikas-Dalikas** [comment]
why this idea was abandoned?
